src/Core/src/Hosting/LifecycleEvents/AppHostBuilderExtensions.iOS.cs

```diff
@@ -19,11 +19,13 @@ static void OnConfigureLifeCycle(IiOSLifecycleBuilder iOS)
 					.OnPlatformWindowCreated((window) =>
 					{
 						window.GetWindow()?.Created();
+						KeyboardAutoManagerScroll.Init();
```

Comment:
Can we call this `Connect` and the other one `Disconnect` ? I think that's more consistent with other parts of the app